pkg/quarterdeck/db/models/users_test.go

```diff
@@ -524,3 +525,69 @@ func (m *modelTestSuite) TestUserPermissions() {
 	require.NoError(err, "could not fetch permissions for user")
 	require.Len(permissions, 18, "wrong number of permissions, have the owner role permissions changed?")
 }
+
+func (m *modelTestSuite) TestListUsers() {
+	require := m.Require()
+
+	ctx := context.Background()
+	orgID := ulid.MustParse("01GQFQ14HXF2VC7C1HJECS60XX")
+
+	//test passing null orgID results in error
+	users, cursor, err := models.ListUsers(ctx, ulids.Null, nil)
+	require.ErrorIs(err, models.ErrMissingOrgID, "orgID is required for list queries")
+	require.NotNil(err)
+	require.Nil(cursor)
+	require.Nil(users)
+
+	_, _, err = models.ListUsers(ctx, orgID, &pagination.Cursor{})
+	require.ErrorIs(err, models.ErrMissingPageSize, "pagination is required for list queries")
+
+	// Should return all checkers users (page cursor not required)
+	users, cursor, err = models.ListUsers(ctx, orgID, nil)
+	require.NoError(err, "could not fetch all users for checkers org")
+	require.Nil(cursor, "should be no next page so no cursor")
+	require.Len(users, 2, "expected 2 users from checkers org")
+	user := users[0]
+	//verify password is not returned
+	require.Empty(user.Password)
+	//verify all other values are returned
+	require.NotNil(user.ID)
+	require.NotNil(user.Name)
+	require.NotNil(user.Email)
+	require.NotNil(user.AgreeToS)
+	require.NotNil(user.AgreePrivacy)
+	require.NotNil(user.LastLogin)
+	require.NotNil(user.OrgID)
+	role, err := user.Role()
+	require.Nil(err)
+	require.Equal("Owner", role)
+	permissions, err := user.Permissions(ctx, false)
+	require.Nil(err)
+	require.Len(permissions, 18, "expected 10 permissions for user")
```

Comment:
Should this error message say "expected 18 permissions for user" ?

Reply:
good catch!  I have made the change.